libkbfs/prefetcher.go

```diff
@@ -203,9 +216,10 @@ func (p *blockPrefetcher) applyToParentsRecursive(
 // hit get marked complete and deleted.
 // TODO: If we ever hit a lower number than the child, panic.
 func (p *blockPrefetcher) completePrefetch(
-	numBlocks int) func(kbfsblock.ID, *prefetch) {
+	numBlocks int, numBytes uint64) func(kbfsblock.ID, *prefetch) {
```

Comment:
Maybe add an explanation of `numBytes` in the comment? I get what it does now but it took a minute of looking over the code which meant I forgot where I came from to look at what it meant in the first place.

Reply:
Done, thanks.